catalyst/utils/metrics/mrr.py

```diff
@@ -0,0 +1,46 @@
+"""
+MRR metric.
+"""
+
+import torch
+
+
+def mrr(outputs: torch.Tensor, targets: torch.Tensor, k=100) -> torch.Tensor:
+
+    """
+    Calculate the MRR score given model ouptputs and targets
+    Users data aggreagtesd in batches.
+    Args:
+        outputs (torch.Tensor):
+            Tensor weith predicted score
+            size: [batch_size, slate_length] 
+            model outputs, logits
+        targets (torch.Tensor):
+            Binary tensor with ground truth. 
+            1 means the item is relevant
+            for the user and 0 not relevant
+            size: [batch_szie, slate_length]
+            ground truth, labels
+
+    Returns:
+        result (torch.Tensor): 
+            The mrr score for each user.
+            
+    """
+    k = min(outputs.size()[1], k)
+    _, indices_for_sort = outputs.sort(descending=True, dim=-1)
```

Comment:
could we use [torch.topk](https://pytorch.org/docs/stable/generated/torch.topk.html#torch.topk) here?

Reply:
I guess the comment is more relevant to the next part
`true_sorted_by_pred_shrink = true_sorted_by_preds[:, :k]`

Anyway, I might missing the advantage of `torch.topk` over the proposed approach. 
We need to sort predictions by the corresponding indexes of the outputs. Is there is a way in pytorch to sort in that fashion?